aws_jobs/cyhy-data-extract.py

```diff
@@ -398,6 +400,11 @@ def main():
                 "retired": True,
                 "scan_types": True,
                 "stakeholder": True,
+                "report_period": True,
+                "windows": True,
+                "scan_limits": True,
+                "scheduler": True,
+                "init_stage": True,
```

Comment:
Is there a reason why you want the `report_period`, `scheduler` and `init_stage` fields exported?  Those fields are currently the same for every entity.

Reply:
With https://github.com/cisagov/cyhy-feeds/pull/52/commits/e1f73a9d5398388127b376820c50e66bf0f21a18, these fields are no longer explicitly mentioned, but the question above still stands.

Reply:
We are trying to maintain data between the two databases, so I was just trying to keep all data in sync

Reply:
If there is no value or requirement to include the `report_period`, `scheduler`, and `init_stage` fields in the exported data, then we should not export them.

Reply:
Looks like with https://github.com/cisagov/cyhy-feeds/pull/52/commits/0622db6f4ce617e202148b31ea343eb80e868261, this conversation can be resolved.